consumer/thor96/README.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Using the Thor96
+permalink: /documentation/consumer/thor96/
+---
+# Using the Thor96
+
+A comprehensive guide to using the [Thor96](https://www.96boards.org/) Consumer Edition development board. This guide is written by the [96Boards](https://www.96boards.org) team at [Linaro](http://www.linaro.org) with community contributions and links to third-party content.
```

Comment:
should Thor96 link point somewhere in particular?

Reply:
Once the website rep is populated I'll change this in another pr